.env

```diff
@@ -65,3 +65,5 @@ SECTION_TITLE_CHAR_LIMIT=200
 
 PROJECT_REDIRECT_HASH_URL=https://s3.amazonaws.com/s3.documentcloud.org/legacy/project_redirects.bin
 ORG_REDIRECT_HASH_URL=https://s3.amazonaws.com/s3.documentcloud.org/legacy/org_redirects.bin
+
+export COMPOSE_FILE=local.yml
```

Comment:
For both the quoting and the addition of the COMPOSE_FILE:
This file is not normally sourced from the command line, it is read by:
https://www.npmjs.com/package/dotenv-flow-webpack
Which does not need the quoting.  The COMPOSE_FILE should export should be in another file made for sourcing.

Reply:
Reverted `.env`